src/components/Wallet/ConnectWallet.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { ConnectWallet, type ConnectWalletProps } from "./ConnectWallet";
+import { WalletProvider } from "../../decorators/WalletProvider";
+
+const meta: Meta<ConnectWalletProps> = {
+  title: "Components/ConnectWallet",
+  parameters: {
+    backgrounds: {
+      default: "container",
+    },
+  },
+  decorators: [
+    WalletProvider,
+    (Story) => <div className="ink-min-h-[300px]">{Story()}</div>,
```

Comment:
The popup doesn't take space, so I added some 😓

Reply:
Likely because it's absolute?

Reply:
It should be absolute to "float" over the UI

Reply:
Yep